### PR TITLE
Move colors from ThemeWrapper into separate file

### PR DESCRIPTION
**Pull request by yukiej**
The colors are all from styles/colors.scss

E2E test: https://builds.cask.co/browse/IT-UPD259
Rat check build: https://builds.cask.co/browse/CDAP-UDUT337

### CODE REVIEW

File: cdap-ui/app/cdap/components/ThemeWrapper/index.tsx [hunk 1]
```diff
@@ -30,24 +31,9 @@ export default class ThemeWrapper extends React.PureComponent<IThemeWraperProps>
...
-        200: '#999999',
-        300: '#bbbbbb',
-        400: '#cccccc',
-        500: '#dbdbdb',
-        600: '#eeeeee',
-        700: '#f5f5f5',
-      },
+      blue,
+      grey,
+      green,
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/ThemeWrapper/index.tsx:45]: If we have moved all the colors can we just add them to our theme (red, bluegrey, orange etc.,)

**Review by ajainarayanan — COMMENTED** [review]
@yukiej Is the change to remove the usage of references to `colors.scss` happening in a separate PR? Ideally we should use the colors from themewrapper instead of colors.scss. Will give us a single source of truth.

Minor comments. Otherwise 👍 LGTM

File: cdap-ui/app/cdap/components/ThemeWrapper/index.tsx (hunk 1 above)
Comment (yukiej) [reply cdap-ui/app/cdap/components/ThemeWrapper/index.tsx:45]: Done!

File: cdap-ui/app/cdap/components/ThemeWrapper/index.tsx [hunk 2]
```diff
@@ -17,6 +17,7 @@
 import * as React from 'react';
 import MuiThemeProvider from '@material-ui/core/styles/MuiThemeProvider';
 import createMuiTheme, { ThemeOptions, Theme } from '@material-ui/core/styles/createMuiTheme';
+import { blue, grey, green, red, bluegrey, orange, yellow, white } from 'colors';
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/ThemeWrapper/index.tsx]: Did you mean `import {...} from 'components/ThemeWrapper/colors`?

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM

**Comment by yukiej** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-UDUT337-1

**Pull request merged**